backend/Controllers/CommentController.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+using AutoMapper;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Http;
+
+using OpenData.Domain.Services;
+using OpenData.Resources;
+using OpenData.Domain.Models;
+
+namespace OpenData.Controllers
+{
+    [Authorize]
+    [Route("/api/[controller]")]
+    public class CommentController : Controller
+    {
+        private readonly ICommentService commentService;
```

Comment:
Reccomend using the `_` prefix for private variables - this is C# standard, and lets us avoid using `this.`